sonar-plugin-api/src/main/java/org/sonar/api/measures/CoreMetrics.java

```diff
@@ -53,6 +60,12 @@
   public static String DOMAIN_SCM = "SCM";
   public static String DOMAIN_ISSUES = "Issues";
   public static String DOMAIN_GENERAL = "General";
+  public static String DOMAIN_DUPLICATIONS = "Duplications";
+
+  /**
+   * @deprecated in 5.5. Renamed to {@link #DOMAIN_DUPLICATIONS}
+   */
+  @Deprecated
   public static String DOMAIN_DUPLICATION = "Duplication";
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS1133)